yscope-k8s/templates/object-store/bucket-creation.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: "batch/v1"
```

Comment:
We don't need this at the moment since we are launching log-viewer via the package.